plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -336,9 +336,13 @@ class DefaultPsiToDocumentableTranslator(
         private fun parseFunction(
             psi: PsiMethod,
             isConstructor: Boolean = false,
-            inheritedFrom: DRI? = null
+            inheritedFrom: DRI? = null,
+            parentDRI: DRI? = null
         ): DFunction {
-            val dri = DRI.from(psi)
+            val currentDri = DRI.from(psi)
```

Comment:
`currentDri` is only used once in the line below, so it would be better to inline it there